src/components/screens/PostsFeed/PostsFeed.js

```diff
@@ -27,12 +28,13 @@ type Props = {
 
 class PostsFeed extends PureComponent<Props> {
   componentDidMount() {
-    const { requestPosts } = this.props;
-    requestPosts();
+    const { requestPosts, userId } = this.props;
+    requestPosts(userId);
```

Comment:
Do we need a check here if posts were not fetched already, to prevent unnecessary fetching?